libbeat/outputs/elasticsearch/config.go

```diff
@@ -24,9 +24,12 @@ type elasticsearchConfig struct {
 }
 
 type Template struct {
-	Name      string `config:"name"`
-	Path      string `config:"path"`
-	Overwrite bool   `config:"overwrite"`
+	Enabled           bool   `config:"enabled"`
+	Name              string `config:"name"`
+	IgnoreES2xVersion bool   `config:"ignore_es2x_version"`
+	Path              string `config:"path"`
+	PathES2x          string `config:"path_es2x"`
+	Overwrite         bool   `config:"overwrite"`
```

Comment:
That seems more flexible and future proof, but what I struggle with in that version is that the we need the 2.x version to be "on" by default. So the default for `template.versions` would be one entry with the 2.x path. That all seems a bit too magical, I have to think more about it.